Source: Gentoo
Upstream: No.
Reason: Fix some potential insecure conversions

diff -urNp cyrus-sasl-2.1.22.org/pwcheck/pwcheck_getspnam.c cyrus-sasl-2.1.22/pwcheck/pwcheck_getspnam.c
--- cyrus-sasl-2.1.22.org/pwcheck/pwcheck_getspnam.c	1999-08-26 19:22:44.000000000 +0300
+++ cyrus-sasl-2.1.22/pwcheck/pwcheck_getspnam.c	2007-02-03 16:29:34.000000000 +0200
@@ -24,6 +24,7 @@ OF OR IN CONNECTION WITH THE USE OR PERF
 ******************************************************************/

 #include <shadow.h>
+#include <string.h>

 extern char *crypt();
